installing/installing-preparing.adoc

```diff
@@ -0,0 +1,86 @@
+
+If you used {product-title} 3 and want to try {product-title} 4, you need to understand how different {product-title} 4 is. {product-title} 4 weaves the Operators that comprise the program and the operating system that the platform runs on, {op-system-first}, together seamlessly. Instead of deploying machines and configuring their operating systems so that you can install {product-title} on them, the {op-system} operating system is an integral part of the {product-title} cluster. Your cluster deploys the operating system for the cluster machines as part of the program itself. The easiest way to use {product-title} is by letting it make all of the infrastructure and machines that it needs.
+
+Because you need to provision machines as part of the {product-title} cluster installation process, you cannot upgrade an {product-title} 3 cluster to {product-title} 4. Instead, you must create a new {product-title} 4 cluster and migrate your {product-title} 3 workloads to them. For more information about migrating, see link:https://redhat-cop.github.io/openshift-migration-best-practices/01-planning.html[OpenShift Migration Best Practices]. Because you must migrate to {product-title} 4, you can use any type of production cluster installation process to create your new cluster.
+
+[id="installing-preparing-existing-components"]
+=== How many existing components do you want to use with your cluster?
+
+Because of the way that the operating system and platform intertwine, it is far easier to complete an installation where you let the installation program for {product-title} stand up all of the infrastructure. These are called _installer provisioned infrastructure_ installations. In this type of installation, you can provide some existing infrastructure to the cluster, but the installation program deploys all of the machines that your cluster initially needs.
+
+You can deploy an installer-provisioned infrastructure cluster without specifying any customization to the cluster or its underlying machines to xref:../installing/installing_aws/installing-aws-default.adoc#installing-aws-default[AWS], xref:../installing/installing_azure/installing-azure-default.adoc#installing-azure-default[Azure], or xref:../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[GCP]. These installation methods are the fastest way to deploy a production-capable {product-title} cluster.
+
+If you need to perform basic configuration for your installer-provisioned infrastructure cluster, such as the region in your cloud that you deploy the machines to and the instance type for the cluster machines, you can customize an installation for xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[AWS], xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-customizations[Azure], or xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[GCP].
```

Comment:
I would not call out the region as a configuration item here that the user **might** need to configure. The user must always configure the region.